### PR TITLE
docs: Add documentation for responsive table columns

### PR DESCRIPTION
**Pull request by vbudovski**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

Documentation for responsive table columns (#23298)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️ 

- [x] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

-----
[View rendered components/table/index.en-US.md](https://github.com/vbudovski/ant-design/blob/table-responsive-docs/components/table/index.en-US.md)
[View rendered components/table/index.zh-CN.md](https://github.com/vbudovski/ant-design/blob/table-responsive-docs/components/table/index.zh-CN.md)

### CODE REVIEW

**Comment by vbudovski** [comment]
Could use some assistance with the chinese translation please.

File: components/table/index.zh-CN.md [hunk 1]
```diff
@@ -135,6 +135,7 @@ const columns = [
 | fixed | （IE 下无效）列是否固定，可选 `true`(等效于 left) `'left'` `'right'` | boolean\|string | false |
 | key | React 需要的 key，如果已经设置了唯一的 `dataIndex`，可以忽略这个属性 | string | - |
 | render | 生成复杂数据的渲染函数，参数分别为当前行的值，当前行数据，行索引，@return 里面可以设置表格[行/列合并](#components-table-demo-colspan-rowspan) | Function(text, record, index) {} | - |
+| responsive | The list of breakpoints at which to display this column. Always visible if not set. | Breakpoint\[] | - |
```
Comment (afc163) [inline components/table/index.zh-CN.md:138]: It is better to add a link for `Breakpoint[]`.

Comment (yoyo837) [inline components/table/index.zh-CN.md:138]: 响应式 `breakpoint` 配置列表。未设置则始终可见。

**Review by afc163 — APPROVED** [review]

**Comment by yoyo837** [comment]
Is it better to link to related docs instead of code files?

**Comment by vbudovski** [comment]
> Is it better to link to related docs instead of code files?

I couldn't find a spot to link to in the docs. Any suggestions?

**Pull request merged**